scripts/Expand-ExchangeFiles.ps1

```diff
@@ -14,7 +14,7 @@ try {
     $pass = ConvertTo-SecureString $Password -AsPlainText -Force
     $cred = New-Object System.Management.Automation.PSCredential -ArgumentList $UserName,$pass
         
-    Invoke-Command -ScriptBlock {Start-Process cmd.exe '/c c:\exchinstall\Exchange2013-x64.exe /extract:c:\exchinstall /passive' -NoNewWindow -Wait} -ComputerName $ComputerName -Credential $cred
+    Invoke-Command -ScriptBlock {Start-Process cmd.exe '/c c:\Exchangeinstall\Exchange2013-x64.exe /extract:c:\Exchangeinstall\bin /passive' -NoNewWindow -Wait} -ComputerName $ComputerName -Credential $cred
```

Comment:
This is the old expand script, left it for backwards compatibility reasons. I'd ditch it once we are happy with the new QS